__tests__/app.test.js

```diff
@@ -325,13 +336,49 @@ describe("/api/users", () => {
           });
         });
     });
+  });
+});
 
-    test("GET 404: should return status code 404 for invalid path for endpoint", () => {
+describe("/api/articles(topic)", () => {
+  describe("GET", () => {
+    test("GET 200: should return articles filtered by topic if topic query is provided", () => {
+      const topic = "mitch";
       return request(app)
-        .get("/api/nonsense")
+        .get(`/api/articles?topic=${topic}`)
+        .expect(200)
+        .then((response) => {
+          expect(response.body).toBeInstanceOf(Array);
+          response.body.forEach((article) => {
+            expect(article.topic).toBe(topic);
+          });
+        });
+    });
+
+    test("GET 200: should return all articles if no topic query is provided", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then((response) => {
+          expect(response.body).toBeInstanceOf(Array);
```

Comment:
Same as above, however we should have already tested for this and so this test probably auto passed without us adding any code which is a hint that we probably don't need it.